snomed/com.b2international.snowowl.snomed.api.impl/src/com/b2international/snowowl/snomed/api/impl/traceability/SnomedTraceabilityChangeProcessorFactory.java

```diff
@@ -38,7 +38,7 @@
 	@Override
 	public ICDOChangeProcessor createChangeProcessor(final IBranchPath branchPath) throws SnowowlServiceException {
 		// SNOMED CT import is in progress
-		if (isImportInProgress(branchPath) || isReindexInProgress()) {
+		if (isImportInProgress(branchPath) || isReindexInProgress() || isClassifyInProgress(branchPath)) {
```

Comment:
Please remove the isClassifyInProgress condition, that was intentionally removed before in this repository. Unfortunately my cherry picked commit reintroduced it again.